app/javascript/components/Places/NewPlace/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import Form from '../Form'
+import styles from './styles.module.scss'
+import { useNewPlaceMutation } from 'api/hooks/places'
+import { useI18n } from 'components/TranslationsProvider'
+
+const NewPlace = () => {
+  const newEventMutation = useNewPlaceMutation()
+  const history = useHistory()
+
+  const createEvent = async values => {
+    try {
+      const { data: place } = await newEventMutation.mutateAsync(values)
+      console.log(place)
+      history.push(`/places/${place.id}`)
+    } catch (err) {
+      console.warn(err)
```

Comment:
I know you were working by example, but maybe we would better show error before the form?

Reply:
maybe not? `err` is technical and not for user's eyes